types/glob-to-regexp/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for glob-to-regexp 0.4
+// Project: https://github.com/fitzgen/glob-to-regexp#readme
+// Definitions by: whatasoda <https://github.com/whatasoda>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ Note that ES6 modules cannot directly export callable functions.
```

Comment:
You can remove this template comments as they are needed and only clutters the file. They are only used as a guide for creating the type file.